website_sale_hide_price/data/product_snippet_template_data.xml

```diff
@@ -9,7 +9,7 @@
         <xpath expr="//span[hasclass('fw-bold')]" position="before">
             <div
                 id="website_hide_price"
-                t-if="user_id.partner_id.website_show_price and not data.get('website_hide_price')"
+                t-if="website.website_show_price and not data.get('website_hide_price')"
```

Comment:
But isn't this a user decision, not website one?

Reply:
`website_hide_price` and `website_show_price` are very confusing, but well... the product condition is being taken into account here?

Reply:
Yes, data contains product information.